.github/workflows/nessie.yaml

```diff
@@ -1,15 +1,33 @@
 name: Nessie
 on:
+  workflow_run:
+    workflows: ["Security-Check"]
+    types:
+      - completed
+  pull_request_target:
   push:
     branches:
       - master
-  pull_request:
-  workflow_dispatch:
+
 jobs:
   deploy-image:
     name: Build and push Docker image
     runs-on: ubuntu-latest
     steps:
+      - name: Extract branch name
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+        id: extract_branch
+
+      # This step will retry until required check passes
+      # and will fail the whole workflow if the check conclusion is not a success
```

Comment:
I think I understand, but I'd like to make sure and to update the comment with extra info.  This step is intended for use with a `pull_request_target` trigger, right?  It will block... forever-ish... until I unlock the check?

Thanks!

Reply:
Exactly!